app/models/ability.rb

```diff
@@ -44,7 +44,7 @@ def not_signed_in
 
     # can view Commercials of confirmed Events
     can :show, Commercial, commercialable_type: 'Event', commercialable_id: Event.where(state: 'confirmed').pluck(:id)
-    can :show, User
+    can [:show, :create], User
```

Comment:
Why do we need create here? Where would that authentication happen (ie where do we access the create action of user)?